axiom/monitors.go

```diff
@@ -96,6 +136,10 @@ type Monitor struct {
 	// Threshold the query result is compared against, which evaluates if the
 	// monitor acts or not.
 	Threshold float64 `json:"threshold"`
+	// CreatedAt is the time when the monitor was created.
+	CreatedAt time.Time `json:"createdAt"`
+	// CreatedBy is the user who created the monitor.
```

Comment:
Sorry, should have been more specific: Name or ID? Please put that in the GoDoc. E.g. `name of the user who ...` or `id of the user who ...`. Same for the other occurrence, please.